src/exercises/1-Introduction/Readme.md

```diff
@@ -7,35 +7,71 @@ In this exercise we’ll learn basics of React Styleguidist.
 ### The task
 
 1. Explore React Styleguidist interface, try to switch to isolated mode (a button in the top right corner), try to edit code of the examples in the browser.
+1. Add a new `Button.js` file (we already added the example Markdown file) in the `src/exercises/1-Introduction` folder (both, `.js` and `.md`, files should have the same name) and see how it’s automatically added to the style guide (restart the style guide if the new component did not appear automatically).
+1. Edit a component source in the `Button.js` file. See how all examples update every time you save a file.
+1. Edit component’s `PropTypes` definition. See how “Props & methods” section reacts to the updates.
+1. Edit examples in the `Button.md` file, add a new example. See how examples in the browser update.
 
-2. Edit examples in the `Button.md` file, add a new example. See how examples in the browser update.
+## 1.2. Introducing styled-components
 
-3. Edit a component source in the `Button.js` file. See how all examples update every time you save a file.
+If you’re not familiar with [styled-components](https://www.styled-components.com/), here’s the idea in five lines of code:
 
-4. Edit component’s `PropTypes` definition. See how “Props & methods” section reacts to the updates.
+```js static
+import styled from 'styled-components';
 
-5. Add a new component and an example Markdown file in the `src/exercises/1-Introduction` folder (both, `.js` and `.md`, files should have the same name) and see how it’s automatically added to the style guide (restart the style guide if the new component doesn’t appear).
+const Title = styled.h1`
+  color: salmon;
+`;
 
-<details>
- <summary>Solution</summary>
+<Title>Hello components!</Title>;
+```
+
+The `Title` component will render an `h1` tag with an attached CSS that sets the color to `salmon`. Styles look almost identical to regular CSS, but instead of classes we have components.
+
+You can also use object notation instead of tagged template literals:
+
+```js static
+import styled from 'styled-components';
+
+const Title = styled('h1')({
+  color: 'salmon'
+});
+
+<Title>Hello components!</Title>;
+```
 
-A component (`src/exercises/1-Introduction/Pizza.js`):
+**Hint:** [A 5-minute intro to styled-components](https://medium.freecodecamp.org/a-5-minute-intro-to-styled-components-41f40eb7cd55).
 
-```jsx static
-import React from 'react';
-export default function Pizza() {
-  return <h1>Hello, pizza! 🍕</h1>;
-}
+### The result
+
+The result should look like this:
+
+```js noeditor
```

Comment:
By the way we can use `padded` modifier instead of spaces now ;-)

Reply:
Not sure what you mean

Reply:
`js noeditor padded`

Reply:
What does it do?

Reply:
The same as spaces between components.

Reply:
Ah that’s too much magic for my taste :)